.github/workflows/stale.yml

```diff
@@ -0,0 +1,21 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
```

Comment:
Just curious why this is running daily at *1:30am*? 🤔

Reply:
1.30 am in which time zone?

Reply:
it's UTC, we can pick any time though.